docs/read_the_docs/source/changelog/changelog_dev.rst

```diff
@@ -6,6 +6,22 @@ Development Changelog
     :version: v3.3.0
     :released: pending
 
+    .. change::
+        :tags: workflow, bugfix
```

Comment:
Doesn't really seem like a bugfix to me. An existing ResStock user couldn't hit an issue, right?

Reply:
It fixes the impending bug as described above in the PR description, no?

Reply:
Prior to pulling in the latest OS-HPXML, there was no bug. After pulling in the latest OS-HPXML and including this PR, there's no bug. Just seems like a back-end refactoring that won't affect any ResStock users.

Reply:
I understand what you're saying; it's not worth mentioning as a bug if it's technically not merged into the code yet. Can you suggest a tag?

Reply:
Yep, that is my point. I'd call it a refactor.